Sources/TuistKit/Services/ShareService.swift

```diff
@@ -131,29 +138,30 @@ struct ShareService {
             )
         }
 
-        if !apps.isEmpty, apps.allSatisfy({ $0.hasSuffix(".app") }) {
-            let appPaths = try apps.map {
-                try AbsolutePath(
-                    validating: $0,
-                    relativeTo: fileHandler.currentPath
+        let appPaths = try await apps.concurrentMap {
+            try AbsolutePath(
+                validating: $0,
+                relativeTo: try await fileSystem.currentWorkingDirectory()
```

Comment:
Shouldn't the paths here be relative to the `path` directory resolved above? That way we are consistent with the usage of `--path` argument to configure the absolute paths that paths are relative to.